pkg/ignition/service_config.go

```diff
@@ -35,13 +35,17 @@ Description=Ironic Agent
 After=network-online.target
 Wants=network-online.target
 [Service]
-TimeoutStartSec=0
+Environment="HTTP_PROXY=%s"
+Environment="HTTPS_PROXY=%s"
+Environment="NO_PROXY=%s"
+TimeoutStartSec=10
```

Comment:
Why was this value changed? If needed for other reasons, I'd prefer a separate PR for this.

Reply:
ok , I will open a separate PR for this.